hasktorch/test/Torch/Typed/AutogradSpec.hs

```diff
@@ -234,7 +234,7 @@ instance
   , StandardFloatingPointDTypeValidation device dtype
   ) => Apply' (RastriginA a (Proxy device') (Proxy dtype')) (Parameter device dtype '[n]) (Tensor device' dtype' '[]) where
   apply' (RastriginA a _ _) parameter =
-    toDevice @device' . toDType @dtype' . rastriginLayer' (toDependent parameter) a $ natValI @n
+    Torch.Typed.Tensor.toDevice @device' . Torch.Typed.Tensor.toDType @dtype' . rastriginLayer' (toDependent parameter) a $ natValI @n
```

Comment:
Not a huge deal since this is buried in a test spec and not user facing, but the verbosity of the module+function name is starting to hurt readability. Any reason not to import so that the functions can be used directly?

Reply:
we will need to take a better look at this. I did not want to make opinionated decisions about modules, function names, scopes, and imports at this point. the change here was necessary because both the `Tensor` and the `Parameter` module now export `toDevice` and `toDType` functions.

Reply:
Could `hiding` on the import of the Parameter module be used to achieve the same thing? I understand though - am fine with coming back to this later once we decide on modules.

Reply:
yes, hiding `toDevice` when importing `Typed.Parameter` would be a way to handle this.